src/file-selector.ts

```diff
@@ -125,18 +125,25 @@ function fromDataTransferItem(item: DataTransferItem, entry?: FileSystemEntry |
         return (item as any).getAsFileSystemHandle()
             .then(async (h: any) => {
                 const file = await h.getFile();
```

Comment:
I think the issue in that case is `h` is undefined, so it will crash on the this line 127.

Reply:
True. That's the one that can be `null`.

Reply:
> If the item's [kind](https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/kind) property is "file", and this item is accessed in the [dragstart](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragstart_event) or [drop](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/drop_event) event handlers, then the returned promise is fulfilled with a [FileSystemFileHandle](https://developer.mozilla.org/en-US/docs/Web/API/FileSystemFileHandle) if the dragged item is a file or a [FileSystemDirectoryHandle](https://developer.mozilla.org/en-US/docs/Web/API/FileSystemDirectoryHandle) if the dragged item is a directory.
>
> Otherwise, the promise fulfills with null.

Essentially, `h` can be null. The return value of `getAsFileSystemHandle` should always be a promise.